_guide/_pages/interviews/technical.md

```diff
@@ -61,7 +61,7 @@ to produce and defend a desired solution. Preface by saying there’s no right a
 
 Help guide the candidate to the right line of thinking, don’t allow them to get too bogged down in the wrong line of thinking — help them get back on track.
 
-* ["How would you design software to run elevators? Can you talk through a high-level design? How would accommodating different building types change your design?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.tgd47iei6k4w)
+* ["Let's say you and your team are tasked to build a feature or an app that would support photo sharing. How would you approach the design and build for this?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.tgd47iei6k4w)
```

Comment:
**Suggestion:** One thing that this is missing relative to the first question is the addition of a separate factor ("How would accommodating different building types change your design?"), which I've found is a really valuable way to assess adaptability to changing requirements 😄. If there was an analogue for this problem (e.g. photo sharing, then you add video sharing? Not sure) that would be helpful, in my opinion.

Reply:
I wonder if we can capture the same idea with devices (scanning photos from an album, snapping selfies with a phone). Either way, I’m 👍 on moving forward, and we can tweak the language again if we don’t get it 100% right.

Reply:
That's a great point! I'm going to link to that in the other issue so that we can noodle it a bit more. Since we're trialing this question in this next round of interviews, hopefully that'll give us some good ideas about followups.

Reply:
Food for noodling...other things you might do with a photo-sharing app:

- Allow shared albums among groups
- Basic image processing: apply filter, crop, overlay text, etc.
- Tag people in images 
- Build an api to allow interaction with 3rd party tools
- Have smart groups (by location, date, etc.)
- Enable social features such as upvotes or re-sharing
- Scale up when/if the app becomes popular

Reply:
Ooh I like all of those - I've used the scale up option in the past since it's pretty generalizable 😄